app/modules/entities/components/layouts/entity_list_row.svelte

```diff
@@ -0,0 +1,148 @@
+<script>
+  import { isSubEntitiesType } from '#entities/components/lib/works_browser_helpers'
+  import { omitClaims } from '#entities/components/lib/work_helpers'
+  import { loadInternalLink } from '#lib/utils'
+  import ImagesCollage from '#components/images_collage.svelte'
+  import Infobox from './infobox.svelte'
+  import { getContext } from 'svelte'
+
+  export let entity,
+    relatedEntities,
+    showInfobox = true,
+    listDisplay = false,
+    displayUri
+
+  let { claims, label, image, images, pathname, serieOrdinal, subtitle, title, type, uri } = entity
+
+  const mainPropertyType = {
+    author: 'wdt:P50',
+    publisher: 'wdt:P123',
+    genre: 'wdt:P136',
+    series: 'wdt:P179',
+    collection: 'wdt:P195',
+    edition: 'wdt:P629',
+    translator: 'wdt:P655',
+    subject: 'wdt:P921',
+  }
+
+  const layoutContext = getContext('layout-context')
+
+  const prop = mainPropertyType[layoutContext]
+  const hasOnlyOneClaimValue = prop => claims?.[prop]?.length === 1
+  if (prop && hasOnlyOneClaimValue(prop)) {
+    claims = omitClaims(claims, [ prop ])
+  }
+</script>
+<div class="entity-wrapper">
+  <div
+    class="cover"
+    class:hasSubEntities={isSubEntitiesType(type)}
+  >
+    <a
+      href={pathname}
+      on:click={loadInternalLink}
+      {title}
+    >
+      <ImagesCollage
+        imagesUrls={images || [ image.url ]}
+        limit={isSubEntitiesType(type) ? 4 : 1}
+      />
+    </a>
+  </div>
+  <div class="entity-info-line">
+    <div class="entity-title">
+      <h3>
+        <a
+          href={pathname}
+          on:click={loadInternalLink}
+          class="link"
+          {title}
+        >
+          {#if layoutContext === 'serie' && serieOrdinal}
+            {serieOrdinal}.
+          {/if}
+          {title || label}
+        </a>
+      </h3>
+      {#if subtitle}
+        <span class="subtitle">{subtitle}</span>
+      {/if}
+      {#if displayUri}
+        <a
+          href={pathname}
+          target="_blank"
+          rel="noreferrer"
+          class="uri"
+          on:click|stopPropagation={loadInternalLink}
+        >
+          {uri}
+        </a>
+      {/if}
+    </div>
+    {#if showInfobox}
+      <div class="entity-details">
+        <Infobox
+          {claims}
+          bind:relatedEntities
+          {listDisplay}
+          shortlistOnly={true}
+          entityType={type}
+        />
+      </div>
+    {/if}
+  </div>
+</div>
+{#if type === 'work'}
+  <slot name="actions" />
+{/if}
```

Comment:
I think it would look better with the row box taking the full width, and the action button in the slot appearing within that box
![Capture d’écran du 2023-05-01 19-54-42](https://user-images.githubusercontent.com/1596934/235502592-e6d55324-0ef0-4154-8ac9-128637a3b072.png)

Reply:
fiou heavier than i thought, should be done (in 82dc2b39f ) by drop global bg-color and to define it in a new wrapper of the whole EntityListRow component  to include Actions slot.